pkg/monitoring/alloy/templates/monitoring-config.yaml.template

```diff
@@ -104,6 +104,7 @@ verticalPodAutoscaler:
       controlledValues: "RequestsAndLimits"
       maxAllowed: 
         cpu: 500m
+        memory: 10Gi
```

Comment:
I assume the issue is related to the va oom bump of 2 as well?

Reply:
Yes, the oom bump of 2 makes the VPA recommendations go crazy high.

But I consider this fix easier than for mimir, because all alloy-metric pods should manage the same max amount of metrics, so their size should be under control.